model/validation.go

```diff
@@ -838,6 +838,55 @@ func ValidateParentAndDrain(drainTime, parentShutdown *duration.Duration) (errs
 				parentShutdownDuration.Seconds(), drainDuration.Seconds()))
 	}
 
+	if drainDuration > (time.Hour * 1) {
```

Comment:
Maybe define `const` for these, e.g. `MaxDrainDuration` and `MaxParentShutdownDuration`?